libportability-gfx/src/impls.rs

```diff
@@ -1307,7 +1307,13 @@ pub extern "C" fn gfxCreateImage(
                 info.samples,
             ),
             info.mipLevels as _,
-            conv::map_format(info.format).unwrap(),
+            match conv::map_format(info.format) {
+                Some(format) => format,
+                None => {
+                    error!("Attempted to create an image with format {:?}", info.format);
+                    return VkResult::VK_ERROR_OUT_OF_HOST_MEMORY
```

Comment:
Is this some sort of debug error handling?

Reply:
In a hindsight, we better just leave an `expect` there for the format. In my local changes I was using RGBA8 as a hacky replacement, but we don't want that :)